enterprise/cli/testdata/coder_groups_list_--help.golden

```diff
@@ -0,0 +1,77 @@
+Usage: coder [global-flags] <subcommand>
```

Comment:
Something seems wrong here, this is the output for `coder --help`, not `coder groups list --help` (see others as well).

Reply:
Yea, something is up. Working on it

Reply:
The issue was I was not making a fresh `*clibase.Cmd` each time, and their was a step of the test that mutates the cmds.

So now I am using a fresh cmd for each test.

Reply:
Fixed